custom_components/sat/setpoint_adjuster.py

```
@@ -23,7 +23,8 @@ def reset(self):
 
     def force(self, target_setpoint: float) -> float:
         """Force setpoint."""
-        self._current = target_setpoint
+        if target_setpoint > self._current:
+            self._current = target_setpoint
```

Pylint (4.0.2)

[refactor] Consider using 'self._current = max(self._current, target_setpoint)' instead of unnecessary if block

(R1731)